daemon/daemon/daemon.go

```diff
@@ -5,12 +5,25 @@ import (
 	"context"
 
 	"github.com/40acres/40swap/daemon/database"
+	"github.com/40acres/40swap/daemon/lib/lightning/lnd"
 	log "github.com/sirupsen/logrus"
 )
 
 func Start(ctx context.Context, db *database.Database) error {
 	log.Info("Starting 40swapd")
 	// TODO
+	// Connect to LND
+	_, err := lnd.NewClient(
+		ctx,
+		lnd.WithLndEndpoint("localhost:10009"),
+		lnd.WithNetwork(lnd.Regtest),
+		// Remember to fill with the correct paths
+		lnd.WithMacaroonFilePath(""),
+		lnd.WithTLSCertFilePath(""),
+	)
+	if err != nil {
+		return err
+	}
```

Comment:
Until we have the config for LND I would not add this to start, next step would be to set up LND-related access, like fs-level access to macaroons, certs and so.

Reply:
True, done!